kura/org.eclipse.kura.core/src/main/java/org/eclipse/kura/core/data/transport/mqtt/MqttDataTransport.java

```diff
@@ -324,7 +328,7 @@ public synchronized void connect() throws KuraConnectException {
             this.cloudConnectionStatusService.updateStatus(this, CloudConnectionStatusEnum.ON);
 
         } catch (MqttException e) {
-            logger.warn("xxxxx  Connect failed. Forcing disconnect. xxxxx {}", e);
+            logger.warn("xxxxx  Connect failed. Forcing disconnect. xxxxx");
```

Comment:
Why the exception was removed?

Reply:
We are throwing the exception then as a KuraConnectionException